packages/recomponents/src/components/r-toggle/r-toggle.story.js

```diff
@@ -0,0 +1,60 @@
+import {storiesOf} from '@storybook/vue';
+import {action} from '@storybook/addon-actions';
+import {boolean, text} from '@storybook/addon-knobs';
+
+import markdown from './r-toggle.md';
+
+storiesOf('Components.Toggle', module)
+    .add('Component', () => ({
+        props: {
+            label: {
+                default: text('Label', 'Toggle me'),
+            },
+            name: {
+                default: text('Name', 'toggle'),
+            },
+            disabled: {
+                default: boolean('Disabled', false),
+            },
+        },
+        methods: {
+            change: action('change'),
+        },
+        data: () => ({
+            model: false,
+        }),
+        template: `
+            <div class="storybook-center">
+                <r-toggle
+                    v-model="model"
+                    :label="label"
+                    :name="name"
+                    :disabled="disabled"
+                    :value="true"
+                    @change="change"/>
+                <p>Toggle value: {{model}}</p>
+            </div>
+        `,
+    }), {
+        notes: {markdown},
+    })
+    .add('Multiple toggles', () => ({
+        data: () => ({
+            toggleModel: [],
+            options: [
+                {value: '1', label: 'label 1'},
+                {value: '2', label: 'label 2'},
+            ],
+        }),
+        template: `
+            <div class="storybook-center">
+                <r-toggle
+                    v-for="option in options"
+                    v-model="toggleModel"
+                    :label="option.label"
+                />
```

Comment:
I'm not sure I'm getting the idea properly here. What is r-toggle actually supposed to do? It has a *simple* checkbox inside, but we're also having `r-checkbox`, right? If we need a radio button, we always have to have the looping inside the parent component, which basically makes the usage harder. In this scenario, r-toggle would be a wrapper component of radio button (checkbox actually) and a label. The component, in it's current form, has no internal state and helps only in styling a regular checkbox.

Which is okay, but I think we need to plan and think of why and where those components are going to be used. Is there any reason putting effort in doing components that have less impact? I mean not using r-toggle would result in something along the lines:

```html
<div v-for="option in options">
  <input type="radio" :id="option.id" :value="value" v-model="toggleModel" class="proper-radio-class">
  <label :for="option.id">{{ option.label }}</label>
</div>
```
That's surely a semi-code, but the idea is that we save nothing by having this component. I would love to use it as:
```html
<r-toggle :options="options" :type="RCheckbox" v-slot:default="option"> <!--  pass down component/name -->
  or have some fancy slot for custom item {{ option.label }}
</r-toggle>
```

Reply:
There are two motivation points to create this component:

* Originally we had separate components for checkbox and for swtich. 

* In some popular libraries we have similar behaviour - radio, checkbox and switch exist together:
https://vuetifyjs.com/en/components/selection-controls
https://muse-ui.org/#/en-US/form

I think that we need this component because it provides different user experience. Checkbox is a checkbox, toggle is a toggle.

Markup between this components is different and merge two components in one will make it more complicated. Right now we have highlited topic like "why we have two components instead of different css". In different scenario we can start conversation on "why we make our components complicated".

Related multiple items - we can use it in the same way as radio button if we want to use multiple components work together. We already wrapping radios in parent component and I think if we already followed this approach previously we should continue do that to make things consistent.